server/db/interface.go

```diff
@@ -99,6 +131,29 @@ type AccountArchiver interface {
 	PayAccount(account.AccountID, string, uint32) error
 }
 
+// MatchData represents an order pair match, but with just the order IDs instead
+// of the full orders. The actual orders may be retrieved by ID.
+type MatchData struct {
+	ID         order.MatchID
+	Taker      order.OrderID
+	TakerAcct  account.AccountID
+	Maker      order.OrderID
+	MakerAcct  account.AccountID
+	Quantity   uint64
+	Rate       uint64
+	Status     order.MatchStatus
+	Sigs       order.Signatures
+	cachedHash order.MatchID
+}
+
+// MatchArchiver is the interface required for storage and retrieval of all
+// match data.
+type MatchArchiver interface {
+	UpdateMatch(match *order.Match) error
```

Comment:
Note that `order.Match` would need modification to include the epoch ID to store with the match.